cookbooks/exim/files/default/noreply/community

```diff
@@ -0,0 +1,10 @@
+This is an automated response to your email, which was sent to an
+unattended address.
+
+If you were trying to reply to a question, answer or comment on
+the community.openstreetmap.org web site then you should do so using
+the web site as it is not possible to reply by email.
```

Comment:
Is this really true? Are we not going to support mailing list mode? Is that not what the mail receiver stuff is for?

Reply:
Not 100% true. It depends on the type of email sent. Digest and many Notification emails cannot be replied to.
The messages that can be replied to will be use another from email.

Reply:
Right but normal replies to non-digest questions and answers should work so maybe the wording could be better? Guess it would need to be fairly vague though, just "you can't reply to this email so please use the web site" or something?

Reply:
Removed the text a bit. I will improve it more once I see the service running a bit.